### DISCUSSION TITLE
`<gap>` vs `<lacuna>`

### DISCUSSION THREAD

**Discussion by pe-ro** [music-encoding/music-encoding#1515] [Ideas]
I've been following the discussion on TEI-L (see https://lists.psu.edu/cgi-bin/wa?A2=TEI-L;c7424531.2407&S=) regarding the difference between a gap in a **transcription of a source** and a gap in **the source itself**.  Some call for a new element, `<lacuna>`, for the latter case, while others advise a redefinition of the existing `<gap>` element.  Personally, I agree with those who want to tweak `<gap>` since it's easier to implement.  I have a feeling this approach will win out for TEI too, but I'm reluctant to make the change before the TEI folks and risk putting MEI out of compliance with TEI on this.  I'm starting this discussion so we don't lose track of the issue.

**Comment by martha-thomae** [comment]
Hi @pe-ro! I was just going to start the same discussion as I need to encode some lacunae (so gaps but in the original source rather than in the transcription). I was wondering if there has been any updates on this since you opened this discussion? Thank you!

**Comment by pe-ro** [comment]
@martha-thomae, it appears that the TEI discussion resulted in 2 new elements: `<lacunaStart/>` and `<lacunaEnd/>`, both empty elements. They're discussed in the TEI Guidelines at https://www.tei-c.org/release/doc/tei-p5-doc/en/html/TC.html#TCAPMI.

However, this doesn't necessarily close off discussion of `<gap>` vs. `<lacunaStart/>` and `<lacunaEnd/>` in MEI practice. I'm interested in hearing what your take is after you have a look at the TEI Guidelines.

**Reply by martha-thomae** [reply]
Thanks, Perry! I will look into it :)
